pkg/scheduler/framework/types.go

```diff
@@ -808,6 +808,11 @@ func (n *NodeInfo) FilterOutPods(pods []*v1.Pod) []*v1.Pod {
 		return pods
 	}
 	filtered := make([]*v1.Pod, 0, len(pods))
+	podKeys := make(map[string]struct{}, len(n.Pods))
```

Comment:
Let's use some common `sets` library:

```suggestion
	npodKeySet := make(sets.String, len(n.Pods))
```